docs/kratos/social-signin/96_native-apps.mdx

```diff
@@ -0,0 +1,147 @@
+---
+id: native-apps
+title: Social sign-in for native apps
+sidebar_label: Native apps
+---
+
+```mdx-code-block
+import Mermaid from '@theme/Mermaid';
+import CodeFromRemote from "@theme/CodeFromRemote"
+```
+
+# Social sign-in for native applications
+
+## Overview
+
+This page covers how to implement social sign-in for native applications via OIDC and OAUth 2.0. The user interaction looks like
+this:
+
+- The user is presented with a login or registration screen that includes a social sign-in button.
+- The user clicks the social sign-in button. A browser window opens (using
+  [ASWebAuthenticationSession](https://developer.apple.com/documentation/authenticationservices/aswebauthenticationsession) on iOS
+  or [Custom Tabs](https://developer.chrome.com/docs/android/custom-tabs/) on Android).
+- The user authenticates with the identity provider and grants the application access to profile information.
+- The user is redirected back to the application and is logged in.
+
+## The native app authentication flow
+
+From a high level, the native app initializes a login or registration flow and receives the first part of the session token
+exchange code from the Ory Network. After the user performed the social sign-in, the user is redirected back to the native
```

Comment:
```suggestion
From a high level, the native app initializes a login or registration flow and receives the first part of the Session Token
Exchange Code from the Ory Network. After the user performed the social sign-in, the user is redirected back to the native
```

Capitalize because it is an Ory Network specific term?

Do we need to define Session Token Exchange Code as an Ory Network specific term here, or is it clear?

Reply:
I'd not capitalize it as it is not a brand name but just some term.